<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConvertionDialog</class>
 <widget class="QDialog" name="ConvertionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Convertion Options</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/new/MyResources/TVcustom.ico</normaloff>:/new/MyResources/TVcustom.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <widget class="QListWidget" name="listWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="5" colspan="5">
    <widget class="QGroupBox" name="FileFormatGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>450</width>
       <height>55</height>
      </size>
     </property>
     <property name="title">
      <string>File Format</string>
     </property>
     <widget class="QComboBox" name="ExtFormatComboBox">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>20</y>
        <width>61</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="EpisodeFormatComboBox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>81</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QComboBox" name="SeasonDashComboBox">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>31</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="ShowNameComboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QComboBox" name="TitleDashComboBox">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>20</y>
        <width>31</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="TitleFormatComboBox">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>20</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="7" colspan="3">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="TitleGetComboBox"/>
      </item>
      <item>
       <widget class="QComboBox" name="TitleSearchComboBox"/>
      </item>
      <item>
       <widget class="QCheckBox" name="AutoTitleCheckBox">
        <property name="text">
         <string>Auto Get Show Title</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="OnlineShowCheckBox">
        <property name="text">
         <string>Use Online Show Names</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="9">
    <widget class="QPushButton" name="CancelPushButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="AddPushButton">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="RemovePushButton">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="8">
    <widget class="QPushButton" name="OKPushButton">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QCheckBox" name="SpacerCheckBox">
        <property name="text">
         <string>Convert &quot;.&quot; to &quot; &quot;</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="RemoveDashCheckBox">
        <property name="text">
         <string>Remove &quot;-&quot;</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="UnderScoreCheckBox">
        <property name="text">
         <string>Convert &quot;_&quot; to &quot; &quot;</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="RemoveYearCheckBox">
        <property name="text">
         <string>Remove Year</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="RemoveBracketsCheckBox">
        <property name="text">
         <string>Remove (), {},[]</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="RemoveExtraJunkCheckBox">
        <property name="text">
         <string>Remove Extra Info</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="SeasonOffsetLabel">
        <property name="text">
         <string>Season Offset</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="SeasonOffsetSpinBox"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="EpisodeOffsetLabel">
        <property name="text">
         <string>Episode Offset</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="EpisodeOffsetSpinBox"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
